
public class ThrowingException {

	public static void main(String[] args) {
		
		Person person = new Person(12);
		
		try {
			person.setAge(-11);
			
			System.out.println("Възраст: " + person.getAge());
		} catch (IllegalArgumentException exception) {
			System.out.println("Възникна грешка: " + exception.getMessage());
		}

	}

}
